sound/soc/intel/common/soc-acpi-intel-mtl-match.c

```diff
@@ -507,6 +526,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_mtl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-mtl-rt713-l0-rt1316-l12-rt1713-l3.tplg",
 	},
+	{
+		.link_mask = GENMASK(2, 0),
+		.links = mtl_rt713_l0_rt1316_l12,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-mtl-rt713-l0-rt1316-l12.tplg",
+	},
```

Comment:
don't we also need a machine driver DMI quirk with the SKU and the relevant JD setting?

Reply:
> don't we also need a machine driver DMI quirk with the SKU and the relevant JD setting?

@plbossart The JD source is RT712_JD1 which is the default value. [unsigned long sof_sdw_quirk = RT711_JD1;](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/intel/boards/sof_sdw.c#L18), and RT711_JD1 = RT712_JD1. So, no quirk is required.